dotenv/main.py

```diff
@@ -252,14 +253,21 @@ def find_dotenv(filename='.env', raise_error_if_not_found=False, usecwd=False):
     return ''
 
 
-def load_dotenv(dotenv_path=None, stream=None, verbose=False, override=False):
+def load_dotenv(dotenv_path=None,
+                stream=None,
+                verbose=False,
+                override=False,
+                encoding='utf-8'):
     f = dotenv_path or stream or find_dotenv()
-    return DotEnv(f, verbose=verbose).set_as_environment_variables(override=override)
+    return DotEnv(f, verbose=verbose, encoding=encoding).set_as_environment_variables(override=override)
```

Comment:
Have you considered just capturing the additional keyword arguments and passing them to the DotEnv constructor? That way you don't have to hard-code a default encoding in 3 different places.